CSS/styles.css

```diff
@@ -571,3 +571,53 @@
 /* ||| help.html ||| */
 
 /* ||| faq.html ||| */
+/* GENERAL TEXT */
+#faq-container{
+    padding: 1rem 4rem;
+    margin-top: 0px;
+}
+#faq-title{
+    color: var(--MESA-Red);
+    padding-top: 10px;
+}
+.faq-heading{
+    padding-bottom:10px;
+    color:var(--COC-Blue);
+}
+.faq-section{
+    padding-top: 30px;
+}
+.faq-question{
+    padding-bottom: 20px;
+}
+
+/* REQUIREMENTS */
+#faq-require-container{
+    padding: 5px;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-items: stretch;
+}
+.faq-require-section{
+    padding-top: 5px;
+    padding-right: 30px;
+    padding-left: 30px;
+    flex-direction: column;
+}
```

Comment:
- `flex-direction: column` has no effect because `.faq-require-section` isn't a flex container.
- Instead of using padding to separate flex items, you can use the flex container property `gap` to define space between flex items
- Also a good habit to get good with shorthand padding properties